doc/policies/authentication.rst

```diff
@@ -182,9 +186,9 @@ emailtext
 type: string
 
 This is the text that is sent via Email to be used with Email Token. This
-text should contain the OTP value.
+text should contain the OTP value *<otp>*.
```

Comment:
"It should contain the OTP value" is basically  right. "the OTP value *<otp>*" is somehow incorrect, since *<otp>* is not the otp value but the tag.
It would be more clear to say

    This text should contain the tag *<otp>* or *{otp}* that is replaced with the actual OTP value.

Or only

    This text should contain the tag *<otp>* or *{otp}*.

Or leave it as is and add the information below at the tags, that the tag *{otp}* will actually fill in the otp value.

Reply:
You are right, I thought the {otp} tag is not used in the message but only <otp>. That's why I added the notation.

But it is added here:
https://github.com/privacyidea/privacyidea/blob/5c265bba65258a24abde5c918ea4396ba41fca79/privacyidea/lib/tokens/emailtoken.py#L439

I favor adding both tag types to the list.

Reply:
In the long run I would probably would like to get rid of ``<otp>``. Or: If I would design it anew, I would not use ``<otp>`` anymore. So I would rather only add ``{otp}`` to the tag list.